src/quickgui/plugin/qgsquickphotopanel.qml

```diff
@@ -27,14 +27,17 @@ Drawer {
   property int iconSize: photoPanel.width/20
   property var fieldItem
 
-  property color bgColor: "white"
-  property real bgOpacity: 0.8
+  property color bgColor: "black"
```

Comment:
should this be in config too?

Reply:
I would keep it for time-being. PhotoPanel could have its own group in qgsquickfeatureformstyling which can be added later. It could contain properties such as icons, coloring, layout of buttons.